src/main/config/application.yml.ctmpl

```diff
@@ -67,10 +67,10 @@ environments:
       password: {{$orsp.Data.crowd_password}}
       interval: 500
       range: 10
-    grails.serverURL: https://orsp.broadinstitute.org/dev
+    grails.serverURL: https://ui.dsp-orsp-dev.broadinstitute.org
     googleSignInClientId: {{$orsp.Data.google_signin_client_id}}
```

Comment:
How are these getting substituted? I don't see that anywhere in this PR.

Reply:
I do not understand this one.

Reply:
The value of `{{$orsp.Data.google_signin_client_id}}` is currently pulled from vault secrets and rendered during the conversion of the ctmpl file to the yml file. I'm guessing that there is a very different mechanism for doing the rendering, but I haven't looked at the latest documentation.

First question: Is this file being used any more? If not, delete it. 
Second question: If this file is being used, where is `$orsp.Data.google_signin_client_id` coming from?

Reply:
got it. Will delete it.